include/cantera/zeroD/ReactorNet.h

```diff
@@ -222,6 +237,20 @@ class ReactorNet : public FuncEval
         return m_integ->maxSteps();
     }
 
+    //! Returns the order used during the last solution step
```

Comment:
I would suggest clarifying that 'order' is the order used by the ODE integrator.

Reply:
noted.